src/Extension/FluentExtension.php

```diff
@@ -161,6 +151,44 @@ public function getLocalisedFields($class = null)
     }
 
     /**
+     * @param string $field
+     * @param string $type
+     * @param string|null $class
+     * @return bool
+     */
+    public function isFieldLocalised($field, $type, $class = null)
+    {
+        if (!$class) {
+            $class = get_class($this->owner);
+        }
+
+        // Data and field filters
+        $fieldsInclude = Config::inst()->get($class, 'field_include');
+        $fieldsExclude = Config::inst()->get($class, 'field_exclude');
+
+        if ($fieldsInclude && $this->anyMatch($field, $fieldsInclude)) {
+            return true;
+        }
+
+        if ($fieldsExclude && $this->anyMatch($field, $fieldsExclude)) {
+            return false;
+        }
+
+        $dataInclude = Config::inst()->get($class, 'data_include');
+        $dataExclude = Config::inst()->get($class, 'data_exclude');
+
+        if ($dataInclude && !$this->anyMatch($type, $dataInclude)) {
+            return false;
+        }
+
+        if ($dataExclude && $this->anyMatch($type, $dataExclude)) {
+            return false;
+        }
+
+        return true;
```

Comment:
I need to re-work this logic... I'm realising it's not working the way I'd invisioned.

Not sure about true as the final default.

Reply:
@mfendeksilverstripe should we just drop `data_exclude`, and have the three configs only? Anything else is excluded.

Reply:
I think we should combine the final check;
return (in data_include) && !(in data_exclude)